lua/smart-splits/mux/wezterm.lua

```diff
@@ -16,7 +18,12 @@ local dir_keys_wezterm_splits = {
 
 local function wezterm_exec(cmd)
   local command = vim.deepcopy(cmd)
-  table.insert(command, 1, 'wezterm')
+  local wezterm_bin_path = config.wezterm_bin_path or 'wezterm.exe'
+  if utils.isWSL() then
+    table.insert(command, 1, wezterm_bin_path)
```

Comment:
The default should be `wezterm` not `wezterm.exe`, until the `isWSL()` block where it should change to `wezterm.exe`

```suggestion
  local wezterm_cli_path = config.wezterm_cli_path or 'wezterm'
  if utils.isWSL() then
    table.insert(command, 1, wezterm_bin_path)
```